.github/workflows/CI.yml

```diff
@@ -120,6 +120,27 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  extra-large-test:
+    # Extra large tests
+    # Only happen during the nightly run, trigerred by cron
```

Comment:
Can we replace the word `nightly` by `daily` or `scheduled` here, as `nightly` might be confused with the julia nightly build?

Reply:
```suggestion
    # Only happens during the daily run, trigerred by cron
```